airflow/config_templates/default_airflow.cfg

```diff
@@ -771,15 +771,17 @@ run_as_user =
 # that allows for the key to be read, e.g. 65533
 fs_group =
 
+# Annotations configuration as a single line formatted JSON object.
+# See the naming convention in:
+#   https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+worker_annotations =
+
+
 [kubernetes_node_selectors]
 # The Key-value pairs to be given to worker pods.
 # The worker pods will be scheduled to the nodes of the specified key-value pairs.
 # Should be supplied in the format: key = value
 
-[kubernetes_annotations]
```

Comment:
given we remove this section, I think we may want to update in the UPDATE.md.

Reply:
updated!